draft-ietf-quic-load-balancers.md

```diff
@@ -1198,6 +1198,87 @@ format and cryptographic protection of this information may change in future
 versions or extensions of TLS or QUIC, and therefore this functionality is
 inherently not version-invariant.
 
+# Interoperability with DTLS over UDP
+
+Some environments may contain DTLS traffic as well as QUIC operating over UDP,
+which may be hard to distinguish.
+
+In most cases, the packet parsing rules above will cause a QUIC-LB load
+balancer to route DTLS traffic in an appropriate way. DTLS 1.3 implementations
+that use the connection_id extension [?I-D.ietf-tls-dtls-connection-id] might
+use the techniques in this document to generate connection IDs and achieve
+robust routability for DTLS associations if they meet a few additional
+requirements.
+
+## DTLS 1.0 and 1.2
+
+DTLS 1.0 [?RFC4347] and 1.2 [?RFC6347] use packet formats that a QUIC-LB router
```

Comment:
DTLS 1.0 isn't getting connection ID support as far as I'm aware.

Reply:
Yes, this is addressing a couple of things:

1) Running DTLS in the same infrastructure as QUIC on the same port. Will a plain old QUIC-LB load balancer break my stuff? The answer is no, except DTLS 1.3 when not using CIDs.
2) #1, plus I want to use Connection IDs to be resistant to address changes. This will just not work for DTLS 1.2 but it will work fine for 1.3 unless the client refuses to use CIDs.